bootstrap/config/crd/bases/bootstrap.cluster.x-k8s.io_ck8sconfigs.yaml

```diff
@@ -87,6 +84,9 @@ spec:
                     - secret
                     type: object
                 type: object
+              channel:
```

Comment:
Why does this change here?
Did we forget running `make generate && make manifests` before?

Reply:
looks like it `make generate && make manifests` created it like that for me